pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -79,6 +79,7 @@ func NewCmdInit(cmdOut io.Writer, parentCommand string) *cobra.Command {
 	flags.Int32VarP(&opts.KarmadaWebhookReplicas, "karmada-webhook-replicas", "", 1, "karmada webhook replica set")
 	flags.StringVarP(&opts.KarmadaAggregatedAPIServerImage, "karmada-aggregated-apiserver-image", "", fmt.Sprintf("swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-aggregated-apiserver:%s", releaseVer.PatchRelease()), "karmada aggregated apiserver image")
 	flags.Int32VarP(&opts.KarmadaAggregatedAPIServerReplicas, "karmada-aggregated-apiserver-replicas", "", 1, "karmada aggregated apiserver replica set")
+	flags.BoolVarP(&opts.KarmadaClusterProxyAdminRBAC, "karmada-cluster-proxy-admin-rbac", "", true, "authorize system:admin to proxy member clusters")
```

Comment:
Can you remind me in what's a situation the user needs to set this flag to false? What's the benefit for him/she?

Reply:
Set this value to `false` if you do not want to proxy access the member cluster on the karmada control plane .

Reply:
That means, the `karmadactl get` command would be completely unavailable, right?

Reply:
Yes, I think so, ask @lonelyCZ to confirm it.

Reply:
I also think it should give `system:admin` highest authority by default. The flag can be removed, but we need to cue users like

https://github.com/karmada-io/karmada/blob/989f3fe8fb88a0f1717dc4853c17c17264f54ebe/artifacts/deploy/cluster-proxy-admin-rbac.yaml#L1-L2

And, we can give the user the instructions in the user manual.

Reply:
Yes, I guess the `system:admin` is the `root` user in a system, it can do anything. So, I doubt if the flag is useful enough. 
Seems we don't have enough evidence, so how about postponing adding this flag until there is a need?

By the way, Is there any document about the `system:admin` user? I just know it's the reserved Kubernetes users, I want to know what's the default permission and how Kubernetes use it.

Reply:
+1

Reply:
Can this parameter be removed?

Reply:
cc @lonelyCZ @RainbowMango

Reply:
> Can this parameter be removed?

+1

cc @GitHubxsy

Reply:
It's OK for me to delete this parameter.

Reply:
Fixed.  @RainbowMango